Makefile

```diff
@@ -64,6 +64,7 @@ test-selenium:
 # exported as a point of reference instead.
 .PHONY: dist
 dist: version-frontend
+	git reset --hard HEAD
```

Comment:
I would suggest doing this in CI only, right before this: https://github.com/rstudio/rsconnect-jupyter/blob/master/.github/workflows/main.yml#L65

Reply:
(since `version-frontend` is part of local workflows, and we don't want to discard local changes in those cases)